src/gui_common/thrive_theme.tres

```diff
@@ -110,52 +110,15 @@ corner_radius_bottom_left = 5
 [sub_resource type="StyleBoxFlat" id=15]
 bg_color = Color( 0.0666667, 1, 0.835294, 1 )
 
-[sub_resource type="StyleBoxLine" id=16]
-content_margin_top = 0.0
-color = Color( 0.439216, 0.623529, 0.623529, 1 )
-thickness = 5
-
-[sub_resource type="DynamicFont" id=17]
-size = 22
-font_data = ExtResource( 1 )
-
-[sub_resource type="StyleBoxEmpty" id=18]
-content_margin_left = 13.0
-content_margin_top = 2.0
-
-[sub_resource type="StyleBoxFlat" id=19]
+[sub_resource type="StyleBoxEmpty" id=16]
 content_margin_left = 13.0
 content_margin_top = 2.0
-bg_color = Color( 0.054902, 0.145098, 0.168627, 0.784314 )
-border_width_left = 1
-border_width_top = 1
-border_width_right = 1
-border_width_bottom = 1
-border_color = Color( 0.0666667, 0.6, 0.537255, 1 )
-corner_radius_top_left = 3
-corner_radius_top_right = 3
-corner_radius_bottom_right = 3
```

Comment:
The theme is now broken. This changes how the options menu looks, how the popups look (including making their title unreadable). You should really review all of the deletions and subresources that are set to null by this PR so that things aren't broken.